src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowSnykScanListenerLS.kt

```diff
@@ -163,11 +169,20 @@ class SnykToolWindowSnykScanListenerLS(
             if (filterTree) {
                 addInfoTreeNodes(rootNode, snykResults.values.flatten().distinct(), fixableIssuesCount)
 
+                val includeIgnoredIssues =
```

Comment:
Moved the filtering of issues to here. That way we don't affect the counts of issues and ignored issues based on the settings.